### ISSUE TITLE
Enhance dropdown menu with aria attributes

### ISSUE THREAD

**Issue by muffinresearch** [mozilla/addons-frontend#3183]
We can enhance the dropdown menu component with additional aria attributes to inform assistive technology of its state e.g. whether it is open or closed. 

An article with some ideas on implementation is here: http://heydonworks.com/practical_aria_examples/#progressive-collapsibles

**Comment by eviljeff** [comment]
If this is your first contribution, please refer to https://wiki.mozilla.org/Add-ons/Contribute/Code on how to get started. 
Mentor @muffinresearch

**Comment by vritant24** [comment]
Hi, I would like to try and implement this feature if it hasn't been taken yet

**Comment by caitmuenster** [comment]
Go for it, @vritant24! If you have any questions, feel free to leave a comment and tag the bug's mentor, @muffinresearch.

**Comment by caitmuenster** [comment]
Hey @vritant24, how's it going with this bug?

**Comment by vritant24** [comment]
I was getting familiar with the codebase, and then I had to take a break to keep up with schoolwork. I will be getting back to it in two weeks. Unless it needs to be fixed asap, then someone else can take it up.

**Comment by caitmuenster** [comment]
Thanks so much! There's no rush -- just wanted to check in and see if you were still interested in working on it. :) Hope all is going well with your schoolwork!

**Comment by vritant24** [comment]
Oh I am definitely still interested in working on it. Just barely getting through with the schoolwork hah.

**Comment by vritant24** [comment]
Alright I am back on this, could I get some help finding where in the repo I would find the code for the dropdown component?

**Comment by kumar303** [comment]
You can create a new branch (with any name) for your changes then push your changes to the branch and create a pull request from there. Here are some detailed instructions: https://services.github.com/on-demand/github-cli/open-pull-request-github

**Comment by vritant24** [comment]
Oh I do have a branch ready, I was just wondering what the base should be. Master?

**Comment by willdurand** [comment]
> Oh I do have a branch ready, I was just wondering what the base should be. Master?

Yes, ideally it should be the `master` branch.

**Comment by tofumatt** [comment]
For QA: if you don't test with assistive technology you can ignore this issue. Otherwise just make sure that the ARIA-aware client is aware of these labels and they're helping, I guess.

**Comment by vcarciu** [comment]
Unfortunately we don't test with assistive technology. Can we mark this as "qa: not needed" ?

**Comment by tofumatt** [comment]
Ah, okay. I've marked it as such then.
